cnd/cnd/javahelp/org/netbeans/modules/cnd/help/CandC++ProjectBasics/AddtlIncludeDirectories.html

```diff
@@ -46,7 +46,7 @@ <h2>
     </ol>
     <hr>
     <p>
-       <small><a href="nbdocs:/org/netbeans/modules/usersguide/legal_notice.htm">Copyright&nbsp;&copy;&nbsp;2015,&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</a></small>
+       <small>
```

Comment:
The HTML is now broken, `<small>` is opened, but not closed. I would remove the `<small>` and `<p>` element. This is present in multiple HTML files.